pages/tidal tools/machine_stats.md

```diff
@@ -124,11 +124,55 @@ dependencies setup:
     my-user@example.com ansible_ssh_private_key_file=path/to/key-file.pem
     ```
 
-4. Execute `machine-stats` and pipe its output to Tidal Tools:
+### Execute Machine Stats Manually
```

Comment:
What do you think if we add a Parent item to aggregate these 2 ways to run machines stats.

It could be something like this.

### How to run machine stats
Now that you have created a hosts file.... you have two options for how to run Machine Stats depending on your needs. For example, you could use machine stats to capture data points on your inventory and send its output to your workspace. Alternatively, You can use Machine Stats to capture statistics on a host for a period of time.  The following 2 sections will guide you through both scenarios.

#### Execute machine stats 

.... your content


#### Run Machine Stats on a Cron Job

.... your content.

Reply:
Makes sense, thanks Mario!